intellij/src/main/java/services/KiotaGenerationLanguage.java

```diff
@@ -0,0 +1,26 @@
+package services;
+
+/**
+ * This is a Enum class representing languages for kiota code generation
+ */
+public enum KiotaGenerationLanguage {
```

Comment:
Consider using `kiota info` to get the list of supported languages automatically from Kiota. This is a backlog task.

Reply:
Tracking with #3110